modules/ROOT/pages/salesforce/salesforce-composite-connector.adoc

```diff
@@ -291,6 +291,197 @@ An example of HTTP input for this flow is:
 }
 ----
 
+== Pre-query Example
+
+This example provides metadata for an object to be queried, generates two sub-requests that contain this metadata, and then executes the sub-requests. The following screenshots show the flow for the Pre-Query example:
```

Comment:
GLOBALLY avoid using a hyphen after prefixes, per the style reference (see the exceptions):
```suggestion
This example provides metadata for an object to be queried, generates two subrequests that contain this metadata, and then executes the subrequests. The following screenshots show the flow for the Pre-Query example:
```